### PR TITLE
WIP - Store weather and location in IDB

### PR DESCRIPTION
**Pull request by agonzalez0515**
##Issue Number:

## Description
Added a package to handle IDB.
Store weather and location in IDB.
Pull weather and location from IDB when offline.

##Do any new issues need to be created as a result of this PR"?
No

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

🚨Please review the [contribution guidelines](https://github.com/gwg-women/gwg-women-techmakers/wiki/Contribution-Guidelines) to this repository.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Created a new branch from 'dev' and PR is targeting the project's 'dev' branch.
- [ ] The issue can be closed.

 Provide a descriptive branch name and once the pull request is approved and merged to dev, the branch can be deleted.

💔Thank you!

### CODE REVIEW

**Review by sonalikatara — APPROVED** [review]

File: src/components/Header.js [hunk 1]
```diff
@@ -82,20 +86,39 @@ class HeaderContainer extends Component {
...
+    }
+
     const {currentCity, currentWeather} = this.state;
     const message = (this.state.currentCity && this.state.currentWeather)
-      ? ` You're in ${currentCity}. It is currently ${currentWeather}°F`
-      : ``;
-
+    ? `You're in ${currentCity}. It is currently ${currentWeather}°F`
+    : 
+    `It is currently ${currentWeather}°F. You're in ${currentCity}.`;
```
Comment (digilou) [inline src/components/Header.js]: Is this just to let us know it's working?

Comment (agonzalez0515) [reply src/components/Header.js]: Oh yea, I had that as a sort of debugging to know when it was loading offline stuff. I can remove it to clean it up.

**Comment by agonzalez0515** [comment]
@sonalikatara @digilou Just realized I need to clean up the idb, please don't merge yet.

**Comment by agonzalez0515** [comment]
Closing this request as I'm switching from the dexie package back to Jake's IDB package that we used in wittr.

**Pull request closed without merging**